docs/api/rest_api/rest_api_reference/input/ibexa-ai.raml

```diff
@@ -1,5 +1,5 @@
-displayName: AI Actions
-/execute/{actionConfiguration}:
+displayName: AI actions
```

Comment:
@mnocon 
Out of sheer curosity, does the change mean that former route ceases to work? 
We have published the previous route in the REST API for 4.6, which means there could be customers already using it.
Doesn't making this change in code without leaving the previous as was constitute a backward compatibility break?